### DISCUSSION TITLE
Import de cantines avec valeurs d'achat

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1599] [Question / réponse]
Lors d'un import de cantines, la base de données refuse les cantines dont le montant total d'achats est renseigné, mais dont l'un au moins des deux autres champs – bio et durable – n'est pas renseigné (cf. captures d'écran en PJ). Il faut que les trois champs ou qu'aucun des champs soit renseigné pour que la base de données accepte l'import. Je suppose que c'est pour éviter d'avoir des valeurs d'achats à partir desquelles la base de données ne peut pas calculer de pourcentages. Mais cela m'astreint soit à mettre "0" dans les champs "bio" et "durable" (ce qui est vraisemblablement faux), soit à supprimer la valeur renseignée pour "total" (ce qui nous fait perdre une donnée).

=> Voulez-vous conserver cette contrainte, @JenniferStephan, @alemangui et @hfroot ? Si oui, laquelle des deux options ("0" ou montant supprimé) privilégier?

<img width="836" alt="Capture d’écran 2022-07-07 à 11 34 50" src="https://user-images.githubusercontent.com/93709421/177744219-8aacd810-3172-4348-95a4-ce279cf34867.png">
<img width="209" alt="Capture d’écran 2022-07-07 à 11 43 37" src="https://user-images.githubusercontent.com/93709421/177744229-62760290-7ea9-4eb0-9423-8e1a79863976.png">

**Comment by hfroot** [comment]
Dans ces cas alors, l'utilisateur.rice n'a pas les données pour les montants bio et/ou durable ?

**Reply by ghost** [reply]
Oui, c'est ça : ils ont le montant total, mais pas le montant de bio ni de durable.
